src/vulkan/pipeline.h

```diff
@@ -114,6 +121,10 @@ class Pipeline {
   PipelineType pipeline_type_;
   std::vector<DescriptorSetInfo> descriptor_set_info_;
   std::vector<VkPipelineShaderStageCreateInfo> shader_stage_info_;
+
+  // Keep the information of what and how to conduct push constant.
```

Comment:
Please add: 

// These are applied from lowest index to highest index, so that if address ranges overlap, then the later values take effect.

Reply:
done